AdminCore.Services/Mappings/ContractMapperProfile.cs

```diff
@@ -8,8 +8,10 @@ public class ContractMapperProfile : Profile
   {
     public ContractMapperProfile()
```

Comment:
any tests for this?

Reply:
I don't think there's any tests for mapping profiles currently.

Reply:
More reason to add tests

Reply:
Cool I'll add an issue in Jira to add tests for Mapping profiles.

Reply:
Added tests for contract mapper